packages/react-components/react-datepicker-compat/stories/DatePicker/DatePickerControlled.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
-import { makeStyles, Button } from '@fluentui/react-components';
 import { addDays, DatePicker } from '@fluentui/react-datepicker-compat';
+import { Button, makeStyles } from '@fluentui/react-components';
+import { Field } from '@fluentui/react-components/unstable';
```

Comment:
You'll need to pull from master and change these stories to import `Field` from `@fluentui/react-components` directly instead of from `@fluentui/react-components/unstable` now that the PR to move it to stable has been merged.